node/lib/openshift-origin-node/model/v2_cart_model.rb

```diff
@@ -1385,10 +1385,20 @@ def do_control_with_directory(action, options={})
 
             buffer << out if out.is_a?(String)
             buffer << err if err.is_a?(String)
-
-            raise ::OpenShift::Runtime::Utils::ShellExecutionException.new(
+            # We ignore 'Disk quota exceeded' msg to allow apps at quota to start/stop/restart.
+            # During moves, quota limits are bumped if necessary.  Here, no bump, just ignore.
+            # Since we aren't bumping quota here, no worries about user exploiting limits 
+            # indefinitely, users can only use up to current limit.  This is a convenience so
+            # nothing gets 'stuck'.
+            if %w[start stop restart].include?(action)
+              raise ::OpenShift::Runtime::Utils::ShellExecutionException.new(
+                        "CLIENT_ERROR: Failed to execute: 'control #{action}' for #{path}", rc, out, err
+                    ) if (rc != 0 && !out.include?("Disk quota exceeded") && !err.include?("Disk quota exceeded"))
```

Comment:
The "Disk quota exceeded" string should be defined as a constant preferably. If we change the error message then this code will no longer work.

But this is all old and its extremely unlikely we'll change the error message... so this should be fine.